### ISSUE TITLE
Controller Support

### ISSUE THREAD

**Issue by dinkuu** [ange-yaghi/engine-sim#434]
i think it would be nice to use for example an xbox controler to use brake, gas pedal, clutch.
please consider this

**Comment by frost768** [comment]
I think you can do it with other key mapping software

**Comment by siglr** [comment]
Yes, easy for buttons like gear up, down, etc. But for axis-like controls (throttle, break, clutch), then it's not very possible at the moment. I think throttle would be especially great to have as an axis.
Thanks!
